embedded/cache/lru_cache.go

```diff
@@ -49,6 +51,17 @@ func NewLRUCache(size int) (*LRUCache, error) {
 	}, nil
 }
 
+func (c *LRUCache) Resize(size int) {
```

Comment:
I don't see this method used in the code other than tests, is this a preparation for future changes about live updates of parameters?

Reply:
haven't labeled the PR as WIP,  yes, it's under preparation